pandas/io/json/_json.py

```diff
@@ -851,9 +844,7 @@ def check_keys_split(self, decoded):
         if bad_keys:
             bad_keys = ", ".join(bad_keys)
             raise ValueError(
-                "JSON data had unexpected key(s): {bad_keys}".format(
-                    bad_keys=pprint_thing(bad_keys)
-                )
+                f"JSON data had unexpected key(s): {pprint_thing(bad_keys)}"
```

Comment:
Not sure if pprint_thing is still needed - can you check?

Reply:
I'm unfamiliar with the specifics of `pprint_thing` and where and when it should be used. Could you let me know why you suspect it might no longer be needed, and I'll investigate.

Reply:
pprint_thing is partially left over from py2/py3 compat, but partially for pretty-pretting nested objects.  In tthis case, `bad_keys` is defined a couple lines up and is a str, so pprint_thing is definitely not needed here.

Reply:
Thanks for the clarification. I removed both the import and the call to `pprint_thing` in that module